gdbinit.py

```diff
@@ -34,7 +42,13 @@ def calculate_hash(file_path):
 
 
 def run_poetry_install(dev=False):
-    command = ["poetry", "install"]
+    check_poetry()  # Check if poetry executable exists in PATH
+    poetry_path = shutil.which("poetry")
```

Comment:
I mean I could add it, if you need. But i have tested this locally `shutil.which`  checks all the directories for the binaries of poetry. So i don't think we need to add it? please let me know if there is something wrong with this.

Reply:
`shutil.which` only works if `~/.local/bin` is on the `$PATH`. On your system (and many systems) that's the case. But on a lot of systems that's not the case (like for our Docker images). So this additional check would allow the dependency update flow to work on those systems as well.